### PR TITLE
bg: S3C-3686 remove random delay from main code

### PR DESCRIPTION
**Pull request by vrancurel**
Instead use the feature from the mock. It makes the code more readable.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -49,6 +49,6 @@
   },
   "devDependencies": {
     "mocha": "^3.3.0",    
-    "zookeeper-mock": "^1.1.0"
+    "zookeeper-mock": "^1.1.1"
```
Comment (vrancurel) [inline package.json]: Note the `npm` is not yet created so tests won't pass.

**Comment by miniscruff** [comment]
No issues on this side of things, will re-review when the dependency is approved.

**Review by miniscruff — CHANGES_REQUESTED** [review]
Marking as needs work as it gives me the best notification when a change is made...

**Review by miniscruff — APPROVED** [review]
Approving even though the dependency isn't updated as I already reviewed that one too

**Review by naren-rajendran — APPROVED** [review]

**Review by jonathan-gramain — APPROVED** [review]

**Pull request merged**